core/dbt/source_config.py

```diff
@@ -7,7 +7,7 @@
 
 class SourceConfig:
     AppendListFields = {'pre-hook', 'post-hook', 'tags'}
-    ExtendDictFields = {'vars', 'column_types', 'quoting', 'persist_docs'}
+    ExtendDictFields = {'vars', 'column_types', 'quoting', 'persist_docs', 'docs'}
```

Comment:
Looks like this line failed for pep8 reasons:
```
core/dbt/source_config.py:10:80: E501 line too long (82 > 79 characters)
```

Reply:
Just fixed.

Reply:
Great, thanks! Tests are running now